js/runner.go

```diff
@@ -547,7 +575,8 @@ func (u *VU) runFn(
 		u.Transport.CloseIdleConnections()
 	}
 
-	state.Samples <- u.Dialer.GetTrail(startTime, endTime, isFullIteration, isDefault, stats.IntoSampleTags(&tags))
+	// TODO: stats.NewSampleTags can be cached and not do it for each iteration
```

Comment:
Not if we have `iter` enabled though...

Reply:
yeah, I think I caught on that later on :(

Reply:
fixed in 057f106